content/docs/operating/security.md

```diff
@@ -34,10 +36,10 @@ that service discovery system.
 
 Scraped targets may be run by untrusted users. It should not by default be
 possible for a target to expose data that impersonates a different target.  The
-`honor_labels` option removes this protection, as can certain relabelling
-setups.
+`honor_labels` option removes and certain relabelling setups remove this
+protection.
 
-As of Prometheus 2.0, the `--web.enable-admin-api` flag controls access to the
+As of Prometheus 2.x, the `--web.enable-admin-api` flag controls access to the
```

Comment:
This is correct as written

Reply:
I agree that it's correct, but "as of" is often misunderstood as "at this time" as opposed to "from this time on". The change would make it easier on non-native speakers. We could also phrase it as "Starting from 2.0 [...]".

Reply:
It's as of 2.0, not 2.x.